var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,8 +27,7 @@
 
 class Paraview(Package):
     homepage = 'http://www.paraview.org'
-    url      = 'http://www.paraview.org/files/v5.0/ParaView-v'
-    _url_str = 'http://www.paraview.org/files/v%s/ParaView-v%s-source.tar.gz'
+    url      = 'http://www.paraview.org/files/v5.0/ParaView-v5.0.0-source.tar.gz'
```

Comment:
`spack versions` and `spack checksum` do not work unless you have a URL to start from.